go-controller/pkg/config/utils.go

```diff
@@ -74,6 +80,31 @@ func ParseClusterSubnetEntries(clusterSubnetCmd string) ([]CIDRNetworkEntry, err
 	return parsedClusterList, nil
 }
 
+// ParseFlowCollectors returns the parsed set of HostPorts passed by the user on the command line
+// These entries define the flow collectors OVS will send flow metadata by using NetFlow/SFlow/IPFIX.
+func ParseFlowCollectors(flowCollectors string) ([]HostPort, error) {
+	var parsedFlowsCollectors []HostPort
+
+	collectors := strings.Split(flowCollectors, ",")
+	for _, v := range collectors {
+		host, port, err := net.SplitHostPort(v)
```

Comment:
use https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/util/net.go#L142

Reply:
Thing is I run into a circular dependency. pkg/util/net.go imports itself k8s.io/util/net, which we already do on this file?

Reply:
We agreed on Slack to merge as is, the code is badly organized to refactor along with this change, it warrants its own PR